packages/web3-eth/types/index.d.ts

```diff
@@ -146,20 +146,20 @@ export class Eth {
         callback?: (error: Error, balance: string) => void
     ): Promise<string>;
 
-    getStorageAt(address: string, position: number): Promise<string>;
+    getStorageAt(address: string, position: number | string): Promise<string>;
     getStorageAt(
         address: string,
-        position: number,
+        position: number | string,
```

Comment:
It's possible to pass a ``number``, ``string``, or ``BN`` value as position.

```suggestion
        position: number | string | BN,
```